component/authentication-endpoint/src/main/webapp/smsotp.jsp

```diff
@@ -119,26 +119,18 @@
                                     <div class="row">
                                         <div class="span6">
                                              <!-- Token Pin -->
-                                             <div class="control-group">
-                                                <label class="control-label" for="password">
-                                                Enter the code sent to your mobile phone:</label>
-                                                 <%
-                                                     String order = request.getParameter("order");
-                                                     if(order != null) {
-                                                        if (order.equals("backward")) {
-                                                 %>
-                                                    <b> ******<%= request.getParameter("screenvalue") %></b>
-                                                 <%
-                                                     } else {
-                                                 %>
-                                                    <b><%= request.getParameter("screenvalue") %> ******</b>
-                                                 <%
-                                                    } } else {
-                                                 %>
-                                                 <b><%= request.getParameter("screenvalue") %></b>
-                                                 <% }  %>
-                                                <input type="password" id='OTPcode' name="OTPcode" class="input-xlarge"
-                                                size='30'/>
+                                             <% if (request.getParameter("screenvalue") != null) { %>
+                                              <div class="control-group">
+                                               <label class="control-label" for="password">
```

Comment:
Is screen value showed in the control-label for password? Usually it is shown in a lighter gray color?